libcrypt1.mk

```diff
@@ -0,0 +1,47 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS    += libcrypt1
+LIBCRYPT1_VERSION  := 4.4.16
+
+libcrypt1-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://deb.debian.org/debian/pool/main/libx/libxcrypt/libxcrypt_$(LIBCRYPT1_VERSION).orig.tar.xz
+	$(call EXTRACT_TAR,libxcrypt_$(LIBCRYPT1_VERSION).orig.tar.xz,libxcrypt-$(LIBCRYPT1_VERSION),libcrypt1)
+
+ifneq ($(wildcard $(BUILD_WORK)/libcrypt1/.build_complete),)
+libcrypt1:
+	@echo "Using previously built libcrypt1."
+else
+libcrypt1: libcrypt1-setup gettext
```

Comment:
If gettext is actually linked to libcrypt, it should be a depend in the control file as well! :)